.github/workflows/publish-docs.yml

```diff
@@ -51,28 +53,18 @@ jobs:
         NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }}
         PR_NUMBER: ${{ github.event.number }}
 
-    - name: Add netlify link PR comment
-      uses: actions/github-script@v3
+    - uses: peter-evans/find-comment@v2
```

Comment:
Not actionable on this PR, but I've mentioned something similar before - we should try to find a mechanism to identify if an individually authored action goes stale or falls out of maintenance, so we don't end up scrambling for a replacement in that case.